.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,65 @@
+# Publish archives to PyPI and TestPyPI using GitHub Actions
+
+name: Publish to PyPI
+
+# Only run for pushes to the master branch and releases.
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+  # Runs for pull requests should be disable unless for testing purposes
+  pull_request:
+    branches:
+      - master
```

Comment:
```suggestion
  # Runs for pull requests should be disable unless for testing purposes
  #pull_request:
  #  branches:
  #    - master
```

Reply:
This PR is ready for review. 

I will disable builds for pull requests after this PR is approved. After that, this workflow will be triggered for pushed to master branch (to TestPyPI only) and tagged releases (both PyPI and TestPyPI).